solutions/usaco-1011.mdx

```diff
@@ -63,3 +63,39 @@ public class triangles{
 </JavaSection>
 
 </LanguageSection>
+
+<LanguageSection>
+ 
+<PySection>
+
+```py
+file_in = open('triangles.in')
+data = file_in.read().strip().split('\n')
+n = int(data[0])
+
+vertices = [list(map(int, data[i + 1].split())) for i in range(n)]
+max_area = 0
+
+# Brute force through all possible triangles to find the largest possible area
+for i in range(n):
+	for j in range(n):
+		if i != j:
+			if vertices[j][1] == vertices[i][1]:
+				for k in range(n):
+					if k != j and k != i:
+						if vertices[i][0] == vertices[k][0]:
+							area = abs(vertices[k][1] - vertices[i][1]) * abs(vertices[j][0] - vertices[i][0])
+							max_area = max(max_area, area)
+			elif vertices[j][0] == vertices[i][0]:
+				for k in range(n):
+					if k != j and k != i:
+						if vertices[i][1] == vertices[k][1]:
```

Comment:
maybe combine some of the if statements?

Reply:
How does that look?

Reply:
I think you put two tabs for each indent?

Reply:
Does it look food now?

Reply:
Yep!